features/bootstrap/FeatureContext.php

```diff
@@ -130,6 +130,39 @@ public function iShouldHaveElementAtPosition(string $rowText, int $position): vo
         }
     }
 
+    /**
+     * @Then the element :elementCss should have :value in the :attribute attribute
+     */
+    public function theElementShouldHaveAttribute($elementCss, $value, $attribute)
+    {
+        $element = $this->getSession()->getPage()->find('css', $elementCss);
+        $actualAttribute = $element->getAttribute($attribute);
+
+        if (!strpos($actualAttribute, $value)) {
+            throw new \Exception(sprintf(
+                'Expected the element "%s" has %s="%s", found %s="%s"',
+                $elementCss,
+                $attribute, $value,
+                $attribute, $actualAttribute
+            ));
+        }
+    }
+
+    /**
+     * @Then I should see the image :imagePath in the :elementCss element
+     */
+    public function iShouldSeeTheImage($imagePath, $elementCss)
+    {
+        $this->theElementShouldHaveAttribute($elementCss, $imagePath, 'src');
+
+        $element = $this->getSession()->getPage()->find('css', $elementCss);
+
+        $imageSrc = $element->getAttribute('src');
+
+        $this->visitPath($imageSrc);
+        $this->assertSession()->statusCodeEquals(200);
```

Comment:
Is that the only http code we can have back? What if it's cached or something because we store on an object storage (and probably a CDN in real life)